images/ubuntu/static.yaml

```diff
@@ -0,0 +1,29 @@
+# Builds a "distroless" image based on ubuntu:jammy
+#
+# "static" is a image that can be used to package statically built binaries (go, rust, etc)
+
+build:
+  from:
+    type: docker
+    url: docker://ubuntu:jammy
+  run: |
+    apt-get update && apt-get dist-upgrade -y
+    apt-get install --reinstall --download-only -y ca-certificates base-files netbase tzdata
```

Comment:
You could use `--option=Dir::Cache::Archives=` and set the download dir.  Then you'd know you didn't have anything else matching in there.

```bash
dldir=$(mktemp -d ${TMPDIR:-/tmp}/XXXXXX)
apt-get --reinstall install "--option=Dir::Cache::Archives=$dldir"  --download-only \
    ca-certificates base-files netbase tzdata
ls $dldir
dpkg-deb -xv $dldir/ca-certificates.*deb rootfs
..
```

Reply:
updated PR